Frontend/voitheia-App/src/app/components/about/about-us/about-us.component.html

```diff
@@ -1 +1,70 @@
-<p>about-us works!</p>
+
+<!-- Team Container -->
+<div class="container-fluid text-center text-justify">
```

Comment:
We do not need the "container-fluid" since this page will be included in an HTML structure which already contains a container.